.github/CODEOWNERS

```diff
@@ -11,6 +11,8 @@ tests_common/__init__.py @snowflakedb/snowcli @snowflakedb/nade
 tests_common/conftest.py @snowflakedb/snowcli @snowflakedb/nade
 tests_integration/conftest.py @snowflakedb/snowcli @snowflakedb/nade
 tests_integration/test_utils.py @snowflakedb/snowcli @snowflakedb/nade
+src/snowflake/cli/api/metrics.py @snowflakedb/snowcli @snowflakedb/nade
+tests/api/metrics/ @snowflakedb/snowcli @snowflakedb/nade
```

Comment:
I anticipate some friction in the future with us modifying these files a lot and then requiring cli for review when we are the sole contributors, so I added them to shared ownership to prevent that